src/styles/forms/_inputs.scss

```diff
@@ -13,6 +13,13 @@
     transition: color 0.1s ease;
   }
 
+  .is-required {
+    color: var(--KLabelRequiredAsteriskColor, var(--KInputBorder));
```

Comment:
I wasn't sure because the handling for inlined labels is different and lower in this file we are setting the label color to match the border color:

![image](https://user-images.githubusercontent.com/67973710/230940309-350249e2-2974-483f-b172-6e49bb2ca515.png)

Reply:
Hmm... then maybe at least fallback to something like this?

```scss
var(--KLabelRequiredAsteriskColor, var(--KInputBorder, var(--grey-600)));
```